ios/brave-ios/Sources/PlaylistUI/InterfaceOrientationModifier.swift

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+import UIKit
+
+extension EnvironmentValues {
```

Comment:
suggestion: This is very useful and could be useful for us in future with other UI elements, can we move this to shared component.

Reply:
I'll mark a few items to shift to BraveUI near the end of the dev cycle for this